tests/integration-tests/src/test/java/org/apache/activemq/artemis/tests/integration/cluster/reattach/ReattachTest.java

```diff
@@ -133,18 +134,31 @@ public void testReattachTransferConnectionOnSession() throws Exception {
       locator.setRetryInterval(retryInterval).setRetryIntervalMultiplier(retryMultiplier).setReconnectAttempts(reconnectAttempts).setConfirmationWindowSize(1024 * 1024);
       ClientSessionFactoryInternal sf = (ClientSessionFactoryInternal) createSessionFactory(locator);
       ClientSession session = sf.createSession(false, true, true);
+      ClientSession secondSession = sf.createSession(false, true, true);
```

Comment:
no need to make this change... but I'm creating a second test on top of this one

Your change is going into testReattachTransferConnectionOnSession2().



I will do this as part of the merge and amending this as you.. you will still be the author of the commit.

Reply:
Ah ok, a separate test also makes sense, I wasn't sure as they both effectively test "transferConnection" in "ServerSessionImpl".
Thank you for checking and merging so quickly!